[BUG] NullpointerException when running ossIndexAudit

When running the ossIndexAudit task I get the following NullpointerException:
```

Caused by: java.lang.NullPointerException
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at org.sonatype.gradle.plugins.scan.ossindex.DependencyGraphResponseHandler.getDependenciesMapWithVulnerabilities(DependencyGraphResponseHandler.java:174)
        at org.sonatype.gradle.plugins.scan.ossindex.DependencyGraphResponseHandler.handleOssIndexResponse(DependencyGraphResponseHandler.java:57)
        at org.sonatype.gradle.plugins.scan.ossindex.OssIndexAuditTask.audit(OssIndexAuditTask.java:105)
        ... 123 more
Caused by: java.lang.NullPointerException
        at org.sonatype.gradle.plugins.scan.ossindex.DependencyGraphResponseHandler.hasVulnerabilities(DependencyGraphResponseHandler.java:183)
        at org.sonatype.gradle.plugins.scan.ossindex.DependencyGraphResponseHandler.lambda$hasVulnerabilities$3(DependencyGraphResponseHandler.java:184)
        at org.sonatype.gradle.plugins.scan.ossindex.DependencyGraphResponseHandler.hasVulnerabilities(DependencyGraphResponseHandler.java:185)
        at org.sonatype.gradle.plugins.scan.ossindex.DependencyGraphResponseHandler.lambda$hasVulnerabilities$3(DependencyGraphResponseHandler.java:184)
        at org.sonatype.gradle.plugins.scan.ossindex.DependencyGraphResponseHandler.hasVulnerabilities(DependencyGraphResponseHandler.java:185)
        at org.sonatype.gradle.plugins.scan.ossindex.DependencyGraphResponseHandler.lambda$getDependenciesMapWithVulnerabilities$2(DependencyGraphResponseHandler.java:173)

```
Am using version 2.0.9. Here's my gradle config:
```
ossIndexAudit {
    isDependencyGraph = true
    isPrintBanner = false
}
````

--- a/src/main/java/org/sonatype/gradle/plugins/scan/ossindex/DependencyGraphResponseHandler.java
+++ b/src/main/java/org/sonatype/gradle/plugins/scan/ossindex/DependencyGraphResponseHandler.java
@@ -180,9 +180,13 @@ public class DependencyGraphResponseHandler
       Map<PackageUrl, ComponentReport> response)
   {
     PackageUrl packageUrl = dependenciesMap.get(dependency);
-    return !response.get(packageUrl).getVulnerabilities().isEmpty() || dependency.getChildren().parallelStream()
-        .map(child -> hasVulnerabilities(child, dependenciesMap, response))
-        .collect(Collectors.toList())
-        .contains(true);
+    ComponentReport componentReport = response.get(packageUrl);
+    if (componentReport != null) {
+      return !componentReport.getVulnerabilities().isEmpty() || dependency.getChildren().parallelStream()
+          .map(child -> hasVulnerabilities(child, dependenciesMap, response))
+          .collect(Collectors.toList())
+          .contains(true);
+    }
+    return false;
   }
 }

